### ISSUE TITLE
Error handlers are not called

### ISSUE THREAD

**Issue by tuanna-hsp** [ssut/nestjs-sqs#16]
First, thank you for the awesome lib, it saves me a lot of time handling SQS in Nest.

There is one problem though, that the error handlers are not called at all.
Do you have any intuition on how that might happen?
```typescript
@Injectable()
export class WorkspaceDeletionService {
  constructor(
    @Inject(WINSTON_MODULE_NEST_PROVIDER) private readonly logger: Logger,
    private readonly rollbar: RollbarLogger,
  ) {}

  @SqsMessageHandler(SqsQueueName.WORKSPACE_DELETION_QUEUE)
  public async handleMessage(message: AWS.SQS.Message): Promise<void> {
    try {
      const messageBody: WorkspaceDeletionMessage = JSON.parse(message.Body);
      const workspaceId = messageBody.workspaceId;
      if (workspaceId) await this.deleteWorkspace(workspaceId);
    } catch (error) {
      // Because the error will not be passed to the handler methods, I must resort to using try/catch to manually handle the 
     // application error here
      const errorMessage = `Handle workspace deletion message error: ${error}`;
      this.logger.debug(errorMessage);
      this.rollbar.error(errorMessage);
      throw error;
    }
  }

  @SqsConsumerEventHandler(SqsQueueName.WORKSPACE_DELETION_QUEUE, "error")
  public onError(error: Error, message: AWS.SQS.Message): void {
    this.logger.error(error.toString(), JSON.stringify(message));
  }

  @SqsConsumerEventHandler(SqsQueueName.WORKSPACE_DELETION_QUEUE, "processing_error")
  public onProcessingError(error: Error, message: AWS.SQS.Message): void {
   // This method is not called whatsoever
    this.logger.error(error.toString(), JSON.stringify(message));
  }

  @SqsConsumerEventHandler(SqsQueueName.WORKSPACE_DELETION_QUEUE, "timeout_error")
  public onTimeoutError(error: Error, message: AWS.SQS.Message): void {
   // This method is not called whatsoever
    this.logger.error(error.toString(), JSON.stringify(message));
  }
}
```

**Comment by tuanna-hsp** [comment]
https://github.com/ssut/nestjs-sqs/pull/5

Oh I got it.
So the latest changes have not been release yet

**Comment by tuanna-hsp** [comment]
@ssut Could you publish a new lib version 🙏 
That will be super helpful for me and many others.

Thank you.

**Comment by titobundy** [comment]
@ssut Could you create and publish new release  v1.0.1 for PR #7 ?

Github Actions for this PR was never published to NPM.

Thanks.

**Comment by ssut** [comment]
@tuanna-hsp @titobundy I'll do that soon.

**Comment by ssut** [comment]
I've published v1.1.0 on npm: https://www.npmjs.com/package/@ssut/nestjs-sqs.
